src/main/io/osd.c

```diff
@@ -3177,7 +3179,9 @@ static void osdRefresh(timeUs_t currentTimeUs)
             fullRedraw = false;
         }
         osdDrawNextElement();
+#ifndef USE_MSP_DISPLAYPORT
```

Comment:
This disabled the `displayHeartbeat` when MSP_DISPLAYPORT is compiled in. It even does not have to be enabled to disable